src/components/HistoryTable/historyTableContextReducer.ts

```diff
@@ -9,7 +9,9 @@ type Action =
   | { type: "prevPageColumns" }
```

Comment:
I don’t completely understand what triggers this bug/if it’s even worth dealing with but when you click on on the same build variant from two different versions, it won't jump to commit the second time

https://user-images.githubusercontent.com/47064971/160662041-204e4c3d-eec8-4fd6-9d27-cf73ed77f783.mov

Reply:
Figured it out it was our favorite frenemy. The Apollo cache. Since the version already lived in the cache it wouldn't be processed the same way for some reason. If i HAD to guess there was likely some weird JS comparison behavior going on here. Possibly trying to match on a reference to the value in the apollo cache? https://github.com/khelif96/spruce/blob/1c15e03e19a667669d9a5d11a2c4d279423fa781/src/components/HistoryTable/utils.ts#L64 
Setting the `fetchPolicy` to `network-only` so it would always compare to a fresh value fixed it. There isn't any negative impact for using `network-only` here since we don't ever try to fetch older values on this page. They are cached by the context since we need to process them before using them.

https://user-images.githubusercontent.com/4605522/160713001-ee3a7670-ff90-475d-a8ae-96c55e510ca3.mov